Assets/Scripts/Audio/SFX/SFXController.cs

```diff
@@ -81,6 +80,51 @@ public void PlayUpgradedSound (AudioSource source)
         source.PlayOneShot(beeps[11] , 1f);
     }
 
+    public void PlayRNGCrashNoise (AudioSource source)
+    {
+        //Sometimes we crash into a dead enemy with no existing audio source anymore
+        //so I check if it's null
+        //this is probably bad
+        if (source == null) return;
+        int rand = Random.Range(0, crashSounds.Length);
+        source.PlayOneShot(crashSounds[rand], 4f);
+    }
+
+    public void PlayHitmarkerSound (AudioSource source)
+    {   
+        source.PlayOneShot(combatSounds[2], 2f);
```

Comment:
Please make hit marker sounds and kill confirmed sound their own variables, relying on remembering and getting the correct indices seems gross and error prone

Reply:
https://github.com/jacobmcleman/LD46/pull/30/commits/fb0ede07ee286d0a37be569db8cbbebc6fc351c9